gce-scripts/run_all_scripts.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# This is the first script to run. This script calls
+# all the other pertinent scripts for setting up
+# and kicking off runs.
+
+# ./run_all_scripts.sh <data_file_to_split> <#_of_shards> <shard_output_prepend>
+# Ex: ./run_all_scripts.sh top-1m.nocommas.8.31.2017 100 alexa
+
+# Only the first input argument is required. The other two will default
+# to 10 and shard respectively.
+
+# will split up the file top-1m.nocommas.8.31.2017 into 100 files
+# into a dir called input_files, and all the files will start with
+# alexa_. So the shard files will be alexa000.csv, alexa001.csv
+# etc.
+
+# If any of the scripts fails, this hard fails and tells the user what script
+# went wrong.
+
+
+input_file=$1
+number_of_shards=${2-10}
+output_file_name=${3-shard_}
```

Comment:
Maybe I'm missing it, but what is `shard_` ? I don't see it set anyplace else...